src/comp/comp.py

```diff
@@ -24,48 +28,48 @@ def __repr__(self):
 # Write a list comprehension that creates a list of names of everyone
 # whose name starts with 'D':
 print("Starts with D:")
-a = []
+a = [str.name for str in humans if str.name[0].lower() == 'd']
 print(a)
 
 # Write a list comprehension that creates a list of names of everyone
 # whose name ends in "e".
 print("Ends with e:")
-b = []
+b = [str.name for str in humans if str.name[len(str.name)-1].lower() == "e"]
 print(b)
 
 # Write a list comprehension that creates a list of names of everyone
 # whose name starts with any letter between 'C' and 'G' inclusive.
 print("Starts between C and G, inclusive:")
-c = []
+c = [str.name for str in humans for match in [
+    'c', 'd', 'e', 'f', 'g'] if str.name[0].lower() == match]
 print(c)
 
 # Write a list comprehension that creates a list of all the ages plus 10.
 print("Ages plus 10:")
-d = []
+d = [age.age + 10 for age in humans]
 print(d)
 
 # Write a list comprehension that creates a list of strings which are the name
 # joined to the age with a hyphen, for example "David-31", for all humans.
 print("Name hyphen age:")
-e = []
+e = [str(char.name + '-' + str(char.age)) for char in humans]
```

Comment:
You could also use an f string
```suggestion
e = [f"{str(char.name}-{str(char.age))}" for char in humans]
```
Then you don't need to concatenate